nemo/collections/nlp/modules/common/megatron/layer_norm_1p.py

```diff
@@ -40,7 +40,7 @@ def reset_parameters(self):
             torch.nn.init.zeros_(self.bias)
 
         def forward(self, x):
-            return _fast_layer_norm(x, self.weight + 1, self.bias, self.epsilon)
+            return _fast_layer_norm(x, self.weight + 1, self.bias, self.epsilon, memory_efficient=False)
```

Comment:
why is this hardcoded to False? Instead it can be run by just passing self.memory_efficient as a parameter right?

Reply:
AFAIU memory_efficient=False is the previous implementation of _fast_layer_norm. As non memory efficient implementation has been tested I left it like this.
We can make it switchable but I think it would be beneficial to test it first. What do you think?

Reply:
Sounds good, we can keep it as it is